components/scripts/lib/libs.sh

```diff
@@ -12,41 +12,47 @@ failed_to_load_lib() {
   exit "${_UNEXPECTED_ERROR}"
 }
 
+# shellcheck source=lib/build_scan.sh
+source "${LIB_DIR}/build_scan.sh" || failed_to_load_lib build_scan.sh
+
 # shellcheck source=lib/color.sh
 source "${LIB_DIR}/color.sh" || failed_to_load_lib color.sh
 
-# shellcheck source=lib/exit-code.sh
-source "${LIB_DIR}/exit-code.sh" || failed_to_load_lib exit-code.sh
```

Comment:
0: The order here may be important. I had this loaded early on because the other library scripts depend on the functions in this one.  But since everything is loaded before the real script logic is executed, maybe it's okay.  Mentioning it in case we run into problems (this would be something to look at as a possible cause).

Reply:
I'm running through some tests now to verify functionality. Do you want me to leave this ordering if everything looks normal?

Reply:
Yes. If we can get away with the new ordering, that would be great. It's definitely an improvement.

Reply:
Reordering the files did not seem to have any negative impact based on the tests I ran. I'm not sure if there are any edge cases to consider.

Reply:
Let's go with it for now.